python/openassetio_traitgen/generators/python.py

```diff
@@ -61,7 +61,7 @@ def render_template(name: str, path: str, variables: dict):
         # NB: Jinja assumes '/' on all plaftorms:
         #  https://github.com/pallets/jinja/blob/7fb13bf94443f067c74204a1aee368fdf0591764/src/jinja2/loaders.py#L29
         template = env.get_template(f"python/{name}.py.in")
-        with open(path, "w", encoding="utf-8") as file:
+        with open(path, "w", encoding="utf-8", newline="\n") as file:
```

Comment:
[fix] Update `README.md`

Reply:
Do you mean release notes? Or do we need to document in the README about the line separator used?

Reply:
Sorry was jumping between meetings. Release Notes indeed 🙃